### PR TITLE
Benchmarking workflow initial check-in

### PR DESCRIPTION
**Pull request by kachulis**
@yfarjoun could you approve initial check-in.  Once it's in we can start applying updates here.

### CODE REVIEW

**Comment by yfarjoun** [comment]
should this have a test json? what do you think our standards should be? I was thinking that we should and have womtool validate with input using circle-ci....if you put in a json, I'll setup the CI part.

File: BenchmarkVCFs/BenchmarkVCFs.wdl [hunk 1]
```diff
@@ -0,0 +1,1336 @@
+version 1.0
+
```
Comment (yfarjoun) [inline BenchmarkVCFs/BenchmarkVCFs.wdl]: since this is public, let's stick a blurb about what this does.

**Comment by yfarjoun** [comment]
let's have a readme.md and an (MIT) license

**Comment by kachulis** [comment]
I put a bsd-3 license on the entire repo (it’s what comms uses for the
gatk-workflows repos).  Do you have a preference for one license over the
other?  And do you think it’s necessary to have individual licenses on each
workflow?

**Comment by yfarjoun** [comment]
bsd-3 seems fine to me. no need for license on individual files in that
case.

File: BenchmarkVCFs/BenchmarkVCFs.wdl (hunk 1 above)
Comment (kachulis) [reply BenchmarkVCFs/BenchmarkVCFs.wdl]: added as part of a meta section

**Comment by kachulis** [comment]
I think some form of automated testing is definitely a good idea.  `Womtool validate` at the minimum, IMO; In the long run it would be nice to have more sophisticated testing with plumbing data actually running through the workflows.  I have added a test.json if we want to just start with the easy `womtool validate` option.

**Pull request merged**